### PR TITLE
Cleanup .gitignore file

### PR DESCRIPTION
**Pull request by shulard**
I am targetting this branch, because this is a pedantic branch.

Related to: sonata-project/dev-kit#211
## Subject

Remove plateform specific ignore entries from .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-composer.phar
-.idea
-vendor/
-.DS_Store
-build
 phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
 .php_cs.cache
```
Comment (core23) [inline .gitignore]: Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore

**Comment by core23** [comment]
Please rebase, this will retrigger StyleCI. After that we will merge this. @shulard

**Comment by shulard** [comment]
I've rebased the PR.

**Comment by greg0ire** [comment]
Thanks! The best time to ping us is when the build passes though.

**Comment by greg0ire** [comment]
Hmmmm… the coveralls build was not triggered, it seems.

**Comment by shulard** [comment]
Really strange... I've rebased another time 😄, hope it'll work...

**Comment by shulard** [comment]
Hello,

Is it the same problem than on sonata-project/SonataCommentBundle#59 ? Coveralls is still waiting for report but everything else seems ok.

**Comment by greg0ire** [comment]
There does not seem to be a coveralls project for that at all : https://coveralls.io/github/sonata-project/SonataDoctrineMongoDBBundle

**Comment by greg0ire** [comment]
Wait I had the wrong address it seems : https://coveralls.io/github/sonata-project/SonataDoctrineMongoDBAdminBundle

**Comment by greg0ire** [comment]
I'm re-trigerring the build with a close reopen

**Comment by shulard** [comment]
Hum, seems that coveralls is still waiting for the status...

**Pull request merged**